### ISSUE TITLE
Something interesting that I learned about Pres. Washington

### ISSUE THREAD

**Issue by Anarcho-Bolshevik** [dessalines/essays#120]
Quoting from Daniel Immerwahr’s [*How to Hide an Empire*, ch. 1](https://books.google.com/books?id=GYtbDwAAQBAJ&pg=PT20):

> Washington set his affairs in order, but he remained doubtful about westerners’ political allegiances. His fears were confirmed in the 1790s, when backcountry men in Pennsylvania refused to pay a federal tax on alcohol and threatened armed secession. It was the Boston Tea Party all over again, this time with whiskey. Yet, notwithstanding his own recent leadership of a revolution against the financial machinations of a distant government, Washington’s sympathy for the rebels quickly ran dry. Their opposition, he complained to Jefferson, had “become too open, violent and serious to be longer winked at.”
>
>Once again, Washington rode west across the mountains, this time to quash a rebellion. In the end, the uprising dispersed before Washington’s forces arrived. But the episode remains, as the historian Joseph Ellis has observed, the “first and only time a sitting American president led troops in the field.”

I suppose that it is not quite as fascinating as the time [he ordered his troops to execute, against their will, two of their own friends for mutiny](https://books.google.com/books?id=iMJcCgAAQBAJ&pg=PT99), or the time [he recommended the draft when the bribes weren’t drawing in enough recruits](https://www.smithsonianmag.com/history/myths-of-the-american-revolution-10941835), or the [cruel and unusual punishments that his army could commit against disobedient or unwilling troops](https://books.google.com/books?id=HkNnBQiqD5cC&pg=PA161) (and there were many such troops), but I prefer the first episode since it involved the least violence (by a stroke of luck).

**Comment by dessalines** [comment]
That's a great book BTW! It might be a while until I can add this.

**Comment by bellabardos** [comment]
Once in an information security class we were given a term paper on how to find a person by phone number. I didn't know what to write, so I started searching and found [thepinnaclelist](https://www.thepinnaclelist.com/articles/top-rated-essay-writing-services/) which has everything. Specifically, it shows the pros of the best services for assignment help.

**Comment by RubyCastaneda77** [comment]
Daniel Immerwahr’s How to Hide an Empire offers profound and insightful insight into the history of US imperial ambitions. His research into the countries, places, and people that have been forgotten or ignored by popular histories is thought-provoking and eye-opening. It is an important work for anyone wishing to gain a better understanding of the true story behind America's rise to power throughout history.

**Comment by dessalines** [comment]
Agree, its a very good book. I remember a part in it where he talks about how the population of people living in the US's imperial colonies **outnumbered the people** living in the contiguous US. 

BTW @Anarcho-Bolshevik please submit these as PRs, (or click the edit button on the file), and I'll merge them. But I don't have too much time to manually add these for you.

**Comment by Aceeper2** [comment]
yt
